src/usage/UsagePage.tsx

```diff
@@ -9,19 +9,35 @@ import LimitChecker from 'src/cloud/components/LimitChecker'
 
 // Utils
 import {pageTitleSuffixer} from 'src/shared/utils/pageTitles'
+import {isFlagEnabled} from 'src/shared/utils/featureFlag'
+import OrgHeader from '../organizations/components/OrgHeader'
+import OrgTabbedPage from '../organizations/components/OrgTabbedPage'
```

Comment:
and these could go under the `// components` and maybe with absolute path as that seems to be the pattern (?)

Reply:
>absolute path as that seems to be the pattern

➕

we have aliases in webpack to make absolute paths work. Most of the imports in the codebase follow this pattern.

Here's how you can update your IDE to default to absolute paths: https://blog.jetbrains.com/webstorm/2020/07/configuring-the-style-of-imports-in-javascript-and-typescript/